cfgov/jinja2/v1/test-fixture/index.html

```diff
@@ -125,7 +125,7 @@
     (b) In the <head> with the `defer` and `async` attributes
         set on the <script> tag.
 #}
-<script type="text/javascript" src="{{ static('js/atomic/' + atomic_type + suffix + '.js') }}"></script>
+<script type="text/javascript" src="{{ static('js/routes/on-demand/' + atomic_type + suffix + '.js') }}"></script>
```

Comment:
@jimmynotjim Do we need to revert this change?

Reply:
I would think not. The real on-demand scripts do come from `/routes/on-demand`, only the old legacy pages required being built without common.js. Maybe @sebworks can confirm though.

Reply:
I don't think this file is used and can be deleted.